src/oauth.js

```diff
@@ -0,0 +1,22 @@
+
+oauth = {}
+
+oauth.sendSignedRequest = function(url, callback, req){
+    var body = req.body || null;
+
+    chrome.identity.getAuthToken({interactive: true}, function(token) {
+        var xhr = new XMLHttpRequest(); 
```

Comment:
The rest of the file appears to have a 2-space indent, could you please maintain the same style throughout? Thanks!